core/src/main/resources/reference.conf

```diff
@@ -155,13 +159,19 @@ system {
 
   io {
     # Global Throttling - This is mostly useful for GCS and can be adjusted to match
-    # the quota availble on the GCS API
-    number-of-requests = 100000
-    per = 100 seconds
+    # the quota available on the GCS API
+    throttle {
```

Comment:
Not sure we should change the config structure like this because everyone's existing configs would suddenly not line up. Definitely at least needs to be mentioned in release notes.